3.Tests/UI/StepDefinitions/User/LogoutSteps.cs

```diff
@@ -0,0 +1,40 @@
+﻿using OpenQA.Selenium;
```

Comment:
Remove this file from this PR since you will create another PR for the logout test